.bash_nav

```diff
@@ -7,3 +7,31 @@ alias desk='cd /mnt/c/Users/gsera/Desktop'
 # source bash
 
 alias srcbash='source ~/.bashrc'
+
+# ---This alias effectively creates a bookmarking app---
+# it adds global symlink shortcuts for use with the 'cds' command
+# ex: 'cds studycloud' will run 'cd -P ~/.short/studycloud'
+# to add a shortcut: go to the target directory and run 'cds + <shortcut_name>' or 'cds + .' to use the current directory name
+# to remove a shortcut: run 'cds - <shortcut_name>' (you can also tack on multiple shortcut names separated by spaces)
+# to list your shortcuts: run 'cds .'
+cds() {
+    if [ ! -d ~/.short ]; then
+        mkdir ~/.short
+    fi
+    if [ $# -eq 1 ] && [ $1 = "." ]; then
+        ls ~/.short/
+    elif [ $# -eq 2 ] && [ $1 = "+" ]; then
+        if [ $2 != "-" ]; then
+            ln -s "$PWD" "$HOME/.short/$2"
+        fi
+    elif [ $# -ge 2 ] && [ $1 = "-" ]; then
+        local args=( ${@:2} )
+        eval "rm ${args[@]/#/"~/.short/"}"
+    elif [ $# -eq 0 ] || [ -d $1 ] || [ $1 = "-" ]; then
```

Comment:
If the shortcut doesn't exist, it's likely that the user forgot to type `cd` instead of `cds`. Let's check whether a directory like this exists and `cd` into it to save them some time.
It might also be that the user typed `cds -`, which should return to the last directory, just like `cd -`.
Additionally, the user might have typed `cds` (with no arguments), which should take them to $HOME, just like `cd` with no arguments.
Note that the order of these logical statements is important. We are using short-circuiting to prevent "unary operator expected" errors, which occur when we try to use `$1` even when it might not exist.